tagcompare/compare.json

```diff
@@ -16,6 +16,9 @@
         "browserName": "chrome"
       }
     },
+    "phantomjs": {
+      "enabled": true
+    },
```

Comment:
Under what circumstances would this configuration be altered? Is this [soft coding](http://thedailywtf.com/articles/Soft_Coding)?